### ISSUE TITLE
wouldn't M109 be more appropriate than m104?

### ISSUE THREAD

**Issue by ptrakk** [5axes/Calibration-Shapes#75]
wouldn't "M109 R" be more appropriate than "m104 S" in the tempfantower.py script?

this would wait until the temp gets to the temperature before starting the layers.

**Comment by 5axes** [comment]
I don't think so as generaly we start from the higher temperature to the lower ,  M109 are not usefull because we are already at an higher value so the 104 function doesn't wait anything.
